go.mk

```diff
@@ -39,8 +45,12 @@ $(STATICCHECK): $(GITROOT)/tools/go.mod
 $(GOLICENSER): $(GITROOT)/tools/go.mod
 	$(GO) build -o $@ -modfile=$< github.com/elastic/go-licenser
 
-$(ELASTICPACKAGE): $(GITROOT)/tools/go.mod
-	$(GO) build -o $@ -modfile=$< -ldflags '-X github.com/elastic/elastic-package/internal/version.CommitHash=anything' github.com/elastic/elastic-package
+# FIXME elastic-package requires Go 1.18 runtime to build, but apm-server uses on 1.17.
```

Comment:
Is this the reason to stop using the Go toolchain to install elastic-package?

Reply:
Yes, elastic-package doesn't build due to the 1.18 requirement of a kubernetes-sigs library.

Reply:
Is the [`.go-version`](https://github.com/elastic/elastic-package/blob/main/.go-version) file in elastic-package still used? It says 1.17.6. Also, go.mod says `go 1.17`. Also also, I built the repo with Go 1.17 :smile:

Perhaps when you used `go get github.com/elastic/elastic-package@latest`, you were using a Go 1.18 toolchain and it pulled a too-new version of the k8s dependencies?